de.xwic.appkit.webbase/src/de/xwic/appkit/webbase/editors/mappers/DatePickerMapper.java

```diff
@@ -0,0 +1,114 @@
+/*******************************************************************************
+ * Copyright 2015 xWic group (http://www.xwic.de)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * 		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software 
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and 
+ * limitations under the License.
+ *  
+ *******************************************************************************/
+package de.xwic.appkit.webbase.editors.mappers;
+
+import de.jwic.controls.DatePicker;
+import de.xwic.appkit.core.config.model.EntityDescriptor;
+import de.xwic.appkit.core.config.model.Property;
+import de.xwic.appkit.core.dao.IEntity;
+import de.xwic.appkit.core.dao.ValidationResult.Severity;
+import de.xwic.appkit.core.model.queries.IPropertyQuery;
+import de.xwic.appkit.webbase.editors.ValidationException;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+
+/**
+ * Mapper for the InputBox control.
+ *
+ * @author Aron Cotrau
+ */
+public class DatePickerMapper extends PropertyMapper<DatePicker> {
+
+	public final static String MAPPER_ID = "DatePicker";
+
+	/**
+	 * @param baseEntity
+	 */
+	public DatePickerMapper(EntityDescriptor baseEntity) {
+		super(baseEntity);
+	}
+
+	/* (non-Javadoc)
+	 * @see de.xwic.appkit.webbase.editors.mappers.PropertyMapper#loadContent(de.xwic.appkit.core.dao.IEntity, de.jwic.base.IControl, de.xwic.appkit.core.config.model.Property[])
+	 */
+	@Override
+	public void loadContent(IEntity entity, DatePicker text, Property[] property) throws MappingException {
+		Object value = readValue(entity, property);
+		if(value == null) {
+            return;
```

Comment:
This is not the correct behavior. If the value is `null`, the value in the input field is not changed. But if the value is null, the field content need to be cleared.. not sure the right method, but something like text.setDate(null) or .setText("")...